tachyon/math/polynomials/univariate/univariate_dense_coefficients.h

```diff
@@ -69,8 +71,12 @@ class UnivariateDenseCoefficients {
   }
 
   constexpr static UnivariateDenseCoefficients Random(size_t degree) {
-    return UnivariateDenseCoefficients(
-        base::CreateVector(degree + 1, []() { return F::Random(); }));
+    std::vector v =
+        base::CreateVector(degree + 1, []() { return F::Random(); });
+    if (v[degree].IsZero()) {
+      v[degree] = F::One();
+    }
+    return UnivariateDenseCoefficients(std::move(v));
```

Comment:
Please make this change as a separate commit.

Reply:
Umm may I ask why? I think it is part of enforcing proper polynomial construction. It wouldn't be a bug without this refactoring.

Reply:
Done :)